module/VuFind/src/VuFind/IdentifierLinker/BrowZineFactory.php

```diff
@@ -107,7 +107,8 @@ public function __invoke(
             $search,
             $config,
             $fullConfig['DOIServices'] ?? $this->defaultDoiServices,
-            $fullConfig['ISSNServices'] ?? $this->defaultIssnServices
+            $fullConfig['ISSNServices'] ?? $this->defaultIssnServices,
+            $fullConfig['BestIntegratorLinks'] ?? []
```

Comment:
Maybe worth creating a `$this->defaultBestIntegratorLinks` property for consistency with the other properties and to make the factory more extensible? Probably not really necessary, but why not?

Reply:
Hmm.  My thought here was that we have the [BestIntegratorLinks] config section all enabled by default, even though it's unused unless you also enable `bestIntegratorLink` within [DOIServices].  So that way you can actually disable the values in [BestIntegratorLinks] if you want, and it would fall back to using the default text and icon from `bestIntegratorLink`.  If you think it's better to be consistent here, I can disable the config section by default and create those same defaults in code, but then the user would need a code change to fall back to the `bestIntegratorLink` default text.